### ISSUE TITLE
forward pass

### ISSUE THREAD

**Issue by AliKafaei** [NVlabs/PWC-Net#73]
Hi,
Forward pass (test) of one pair image on GPU (Titan V) takes too much time (5 min). Is it related to code or Cuda does not work well.
Another point is that can I re train this model using my own data?

**Comment by luvegood** [comment]
We haven't tested on TitanV or higher version of CUDA. But the inference time takes less than a second, excluding the model loading time.

**Comment by AliKafaei** [comment]
Ok thanks it solved.
Can I re train the model using my own data?

**Comment by luvegood** [comment]
We didn't provide the training code in pytorch, but you can always try using your own data

**Comment by AliKafaei** [comment]
For Training should I use Loss.py from your flownet implementation?

**Comment by ZikunZhou** [comment]
> Ok thanks it solved.
> Can I re train the model using my own data?

I have the same problem, could you tell me how to solve it? Thanks.

**Comment by AliKafaei** [comment]
yes I retrained the model by my own data and I freeze some layers.
tell me the problem I will help you.

**Comment by ZikunZhou** [comment]
I also find that the forward pass of one pair image on GPU (RTX2080) takes a few minutes （Pytorch version）. How did you solve this problem?

**Comment by ZikunZhou** [comment]
> Hi,
> Forward pass (test) of one pair image on GPU (Titan V) takes too much time (5 min). Is it related to code or Cuda does not work well.
> Another point is that can I re train this model using my own data?

I also find that the forward pass of one pair image on GPU (RTX2080) takes a few minutes （Pytorch version）. How did you solve this problem?

**Comment by AliKafaei** [comment]
yes you can re train the model with your own data.
The forward pass does not take much time. for me (Titan V) it took 30 ms.
your code probably is working on CPU.
